index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500&display=swap" rel="stylesheet">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header>
```

Comment:
- [ ] Please when giving code review, it is just appropriate to phrase your comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think your codebase would be cleaner if you remove unused code block commented out. Kindly remove the above-commented code blocks"*
* This reads as an order: *"Remove comments inside the code blocks to avoid chunks"* 

- [ ] Also, your comment will be more useful if it includes an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*